src/components/VoteCounter.jsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Badge from "@mui/material/Badge";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import Button from "@mui/material/Button";
+import AddIcon from "@mui/icons-material/Add";
+import RemoveIcon from "@mui/icons-material/Remove";
+import BoltIcon from "@mui/icons-material/Bolt";
+import { useEffect } from "react";
+
+import "./VoteCounter.css";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { addVoteArticle, takeVoteArticle } from "../api";
+
+const theme = createTheme({
+  status: {
+    danger: "#e53e3e",
+  },
+  palette: {
+    primary: {
+      main: "#7cbb3c",
+      darker: "#dddddd",
+    },
+    darker: {
+      main: "#5ca373",
+    },
+    neutral: {
+      main: "#b46544",
+      contrastText: "#fff",
+    },
+  },
+});
+
+export default function VoteCounter({ voter, setVoter, article_id }) {
+  const handleAddVote = () => {
+    addVoteArticle(article_id, 1).then((data) => {
+      const votesNum = data.article.votes;
+      setVoter(votesNum);
+    });
+  };
+  const handleTakeVote = () => {
+    takeVoteArticle(article_id, 1).then((data) => {
+      const votesNum = data.article.votes;
+      setVoter(votesNum);
+    });
+  };
```

Comment:
Similarly it would be possible to refactor these two functions into 1 function (named `handleVoteChange` for example), where pressing different buttons invokes it with different numbers of votes to either add or take from the total.

Reply:
Will do Katherine, thanks.